lib/ldp.js

```diff
@@ -147,7 +147,7 @@ class LDP {
     debug.handlers('POST -- On parent: ' + containerPath)
     // prepare slug
     if (slug) {
-      if (this.isAuxResource(slug, extension)) throw error(403, 'POST is not allowed for auxiliary resources')
+      if (this.isAuxResource(slug, extension)) throw error(403, 'POST is not allowed for auxiliari resources')
```

Comment:
```suggestion
      if (this.isAuxResource(slug, extension)) throw error(403, 'POST is not allowed for auxiliary resources')
```
Why the change from correct spelling to incorrect?

Reply:
@TallTed just a fake commit that is reverse to go to the tests that are now on github actions